src/main/java/com/wepay/model/data/AddressData.java

```diff
@@ -26,4 +26,15 @@ public static JSONObject buildAddress(AddressData info) throws JSONException {
 		return o;
 	}
 
+	public static JSONObject buildUnifiedAddress(AddressData info) throws JSONException {
```

Comment:
Should this become the default buildAddress and have the old one be renamed to buildLegacyAddress or something like that? Why would someone on a new SDK want an old address?